test/testsymboldatabase.cpp

```diff
@@ -11354,6 +11356,28 @@ class TestSymbolDatabase : public TestFixture {
         ASSERT(!x->varId());
         ASSERT(!x->variable());
     }
+
+    void structsWithSameName() {
```

Comment:
it seems you just randomly added this test at the end. It would be nice if you could locate where the other tests for SymbolDatabase::findType are. There should be more tests here in this file.